### DISCUSSION TITLE
How to enable experimental discord album art feature?

### DISCUSSION THREAD

**Discussion by DestroyerAce** [Taiko2k/Tauon#1118] [Q&A]
Hi, I saw that the feature mentioned in title was added in the latest(7.7.2) version of TMB but there's no option to enable it in settings.

**Comment by Taiko2k** [comment]
It's always on, no setting, but it doesn't always work since it works by scrapping public data. You should see it work for a popular album. Let me know if it's never working.

**Reply by DestroyerAce** [reply]
I've tried it with some of Kanye and Arctic Monkey's albums but still no cover art. Also, all music files are tagged from Picard.

**Reply by RSKYS** [reply]
Not all tracks got arts depend on metadata, personally some of my tracks have correct tags and yet they still not get fetched from musicbrainz, getting them locally would also have huge impact on performance; means only possible enahcement could be an alt binding webservice.
